velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -886,7 +886,11 @@ std::vector<MaterializedRow> materialize(const RowVectorPtr& vector) {
     MaterializedRow row;
     row.reserve(numColumns);
     for (size_t j = 0; j < numColumns; ++j) {
-      row.push_back(variantAt(vector->childAt(j), i));
+      // variantAt() assumes you can upcast to SimpleVector, so we need to take
+      // the inner vector out of lazies first.
+      const auto& child = vector->childAt(j);
+      row.push_back(variantAt(
+          child->isLazy() ? BaseVector::loadedVectorShared(child) : child, i));
```

Comment:
Can this change be extracted into a separate PR? 

Can we call loadedVectorShared once per column vs. once per column per row?